src/components_js_minifier.erl

```diff
@@ -0,0 +1,57 @@
+-module(components_js_minifier).
+
+-behaviour(cowboy_http_handler).
+
+-export([init/3]).
+-export([handle/2]).
+-export([terminate/3]).
+
+init(_, Req, _Opts) ->
+    {ok, Req, []}.
+
+handle(Req, State) ->
+    {Components, Req1} = cowboy_req:qs_vals(Req),
```

Comment:
Jag funderar ifall vi bör parsa qs-strängen först för att se att component/tag är valid? Det skulle gälla andra entry-points iså fall tänker jag.